### ISSUE TITLE
there is no comment in acme-nginx role

### ISSUE THREAD

**Issue by glatteis** [studieren-ohne-grenzen/sog-ansible#20]
The readme has some misleading information about certbot and nginx (there is no specific comment about how to install nginx / certbot). Of course, the best solution here would be to somehow automatically circumvent the weird behavior, for instance start nginx with  a different config before certbot has run for the first time.

**Comment by marcelklehr** [comment]
I'm not sure the info in the readme applies at all anymore

**Comment by glatteis** [comment]
Yes, by installing nginx first (which failed to start), then running the certbot stuff by copying into the top of the role.
